### PR TITLE
Added the Contributing.md and Code of Conduct File into the Repo

### PR DESCRIPTION
**Pull request by ConnectBhawna**
I have added a Code of conduct along with Contributing guidelines to the project. In my Contributing guideline, I have not provided a link for the IHR Code of conduct, in case this gets merged I would add the link else the link would redirect to my repo. Let me know what you think about the rules and approach I have provided. @thunderbiscuit  sir incase you need some changes let me know I would make and send the PR again.

Solved issue #245

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+$ git branch -d <type>/<issue|issue-number>/{<additional-fixes>}
+$ git push --delete origin <type>/<issue|issue-number>/{<additional-fixes>}
+# Fetch and merge with upstream/master
+$ git checkout master
+$ git pull upstream
+$ git push origin
+```
+
+- Always follow [commit message standards](https://chris.beams.io/posts/git-commit/)
+- About the [fork-and-branch workflow](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/)
```
Comment (thunderbiscuit) [inline CONTRIBUTING.md:86]: Let's remove this part and only keep the top welcome message.

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Hi @ConnectBhawna! Thanks for taking the time to open a PR. Adding a CONTRIBUTING file is a good idea.

I'm reading this over and think the code of conduct might be a little bit too heavy-handed for what I want. I'm thinking of maybe writing one of my own that would be much simpler. Do you mind removing that file?

As for the contributing file, let's start with just the basics. I like that it exists, and I can imaging things we might want to add to it. But basic git workflows I don't think belong there for now (also they're not quite the workflows I would recommend in any case).

**Comment by ConnectBhawna** [comment]
Okk @thunderbiscuit sir let me do this :)

**Comment by thunderbiscuit** [comment]
Perfect thanks! Just two little things: I get notifications when people open issues or PRs on the repo, and when they comment on any of those. No need to tag me in all your messages 🤣. Also, you don't need to call me Sir.

**Comment by ConnectBhawna** [comment]
Added the changes as you mentioned @thunderbiscuit sir :)

**Comment by thunderbiscuit** [comment]
I think you still have 3 commits, and incorrect commit messages. Let me know if you'd like help on how to fix this (see my message above).

You'll need to do an interactive rebase and force-push your new commit. In this case I'm looking for 1 commit, and your commit message will be:
```txt
Docs: Add CONTRIBUTING.md file
```

**Pull request closed without merging**